tests/unit/raf-test.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+import { test, moduleForComponent } from 'ember-qunit';
+import wait from 'ember-test-helpers/wait';
+import { generateContent, sortItemsByPosition } from '../helpers/helpers';
+import template from '../templates/fixed-grid';
+
+moduleForComponent('ember-collection', 'raf', {
+  integration: true,
+});
+
+test('works without requestAnimationFrame', function(assert) {
+  var originalRaf = window.requestAnimationFrame;
```

Comment:
this setup/teardown should be in setup/teardown of the module, otherwise it is possible for a failure to prevent restoration